includes/entity_field_updates.js

```diff
@@ -8,8 +8,8 @@ module.exports = (params) => {
                 nonNull: ["entity_id", "occurred_at", "key_updated", "entity_table_name"]
             },
             bigquery: {
-                partitionBy: "DATE(occurred_at)",
-                clusterBy: ["entity_table_name", "key_updated"],
+                partitionBy: "RANGE_BUCKET(entity_table_name_partition_number, GENERATE_ARRAY(0, 1000, 1))",
```

Comment:
@alexchristensenedu good spot. This was deliberate though. I wanted bucket 1000+ to mean something in some cases (valid_to IS NOT NULL I think) so I wanted at least a gap of 1 bucket between that and the buckets for which valid_to IS NULL. I thought it might reduce the risk of human error due to things like the difference between < and <= or [0] and [1] which are easy to get the wrong way round.